File: libs/contracts/Dto/CountryDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HSMS.contracts.Dto
{
    public class CountryDto
    {
        public int Id { get; set; }

        public string Name { get; set; } = string.Empty;

        public string IsoCode { get; set; } = string.Empty;

        public string PhoneCode { get; set; } = string.Empty;

        public DateTimeOffset CreatedAt { get; set; }

        public DateTimeOffset UpdatedAt { get; set; }
    }
}

File: apps/HSMS-fe/src/app/models/country.dto.ts

export interface CountryMastersDto {
   id: number;
  name: string;
  isoCode: string;
  phoneCode: string;
  createdAt: string;
  updatedAt: string;
}

export interface CountryDto {
  name: string;
  isoCode: string;
  phoneCode: string;
}

export interface ApiResponse<T = any> {
  success: boolean;
  data?: T;
  message?: string;
  errors?: string[];
}